testnet/stacks-node/src/burnchains/bitcoin_regtest_controller.rs

```diff
@@ -638,18 +757,21 @@ impl BitcoinRegtestController {
     ) -> Option<Transaction> {
         let public_key = signer.get_public_key();
 
-        let (mut tx, utxos) = if let Some(utxo) = utxo_to_use {
+        let (mut tx, mut utxos) = if let Some(utxo) = utxo_to_use {
             (
                 Transaction {
                     input: vec![],
                     output: vec![],
                     version: 1,
                     lock_time: 0,
                 },
-                vec![utxo],
+                UTXOSet {
+                    bhh: BurnchainHeaderHash::zero(),
+                    utxos: vec![utxo],
+                },
             )
         } else {
-            self.prepare_tx(&public_key, DUST_UTXO_LIMIT, 1)?
+            self.prepare_tx(&public_key, 3 * DUST_UTXO_LIMIT, None, None, 0)?
```

Comment:
Where does the `3 * DUST_UTXO_LIMIT` constant come from?

Reply:
Refactored in https://github.com/blockstack/stacks-blockchain/pull/2383/commits/4d576c9ffe82b31c8efe4b4659f30b6fa7e576cd